docs/user/YAL5-single-codons_usage.md

```diff
@@ -0,0 +1,99 @@
+# Running YAL5-single-codons.R
+
+### Inputs 
+
+YAL5-single-codons looks into metafeatures, currently specificly codon use. It takes reads from the H5 file of a sample and assigns them to the cooresponding codon position for in a gene using the species GFF3 file and  
+a tsv file listing the codons for all genes of the species being studied, ie the yeast_codon_table.tsv. The script runs to calculate the average relative number of reads mapping to a feature of interest, relative to a window of
+codons adjacent to the feature. The default value of this window is 5 codons either side of the feature of interest. This can be used to study how inhibitory a feature is, as more inhibitory features will have a larger average 
+relative read value. 
+
+
+### Outputs
+
+There are currently two potential outputs when running YAL5-single-codons.R. 
+
+The first is a pdf of a graph plotting the average relative number of reads mapping to each position in the desired window around the feature of interest.
+This is produced if only one feature is given, for example codon 'CGA'.
+
+The second potential output is a table in the form of a tsv file. This is produced when multiple features are given in the form of a tsv file; for example all possible codons. This table will contain the average relative count of 
```

Comment:
Maybe expand this sentence a bit? 

This is produced when multiple features of interest are provided in the form of a tsv file as the input for the feature_of_interest argument